authLdap.php

```diff
@@ -276,6 +285,14 @@ function authLdap_login($user, $username, $password, $already_md5 = false)
 		return $user;
 	}
 
+	// don't do anything when the password is not defined - assume an alternate method of authentication
+	if ($password == null) {
```

Comment:
I would like to keep login-logic out of this method. Can we somehow handle this case in the `Authenticate` class?

Reply:
I'm pretty sure I can just remove that block altogether - I might have left that one in for testing by mistake.